templates/azure-pipelines.yml

```diff
@@ -140,6 +140,10 @@ stages:
           CXX: clang++-3.5
           PACKAGES: clang-3.5
     steps:
+    # TODO: remove
+    - checkout: self
+      submodules: true
```

Comment:
Why are these needed?

Reply:
They were there because you had them in https://github.com/vinniefalco/json/commit/ab93acee16b06978d73e2c2f81b6cebc14280011 and I was thinking "hmm, if the goal is to enable copy-paste and any libs need this, just have them all do it."  But if I'm understanding Mateusz correctly, they are not and I'll remove them.

Reply:
Ahh... yes, I remember now. JSON used to use docca as a submodule. But that is changed now, docca is part of boost (https://github.com/boostorg/docca). I believe you can remove those.

Reply:
You can remove those lines, please

Reply:
Hi, I'm sorry for the delay :(  I removed the recursive cloning.